.github/workflows/tests.yml

```
@@ -9,12 +9,42 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
-      # Install dependencies, cache them correctly
-      # and run all Cypress tests
+      # Get path of cache 
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "dir=$(yarn cache dir)" >> $GITHUB_OUTPUT
+      
+      - uses: actions/cache@v3
+        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
```

```suggestion
          path: |
            ${{ steps.yarn-cache-dir-path.outputs.dir }}
            ${{ github.workspace }}/.next/cache
```

This works for caching the Yarn install directory (yay!), but we need to make it do the Next cache as well. Based on the docs, I think you just need to add `${{ github.workspace }}/.next/cache` here, but experiment and see what works.

---

You're right, I forgot to add that line after implementing for yarn instead of npm

---

No worries, the documentation is super unclear for Yarn (I think Vercel has an anti-yarn vendetta)